{
  "tests/aws/services/backup/test_backup.py::TestBackup::test_backup_plan": {
    "recorded-date": "11-04-2024, 23:50:41",
    "recorded-content": {
      "create-backup-plan": {
        "BackupPlanArn": "arn:<partition>:backup:<region>:111111111111:backup-plan:<backup-plan-id:1>",
        "BackupPlanId": "<backup-plan-id:1>",
        "CreationDate": "<datetime>",
        "VersionId": "<version-id:1>",
        "ResponseMetadata": {
          "HTTPHeaders": {},
          "HTTPStatusCode": 200
        }
      },
      "get-backup-plan": {
        "BackupPlan": {
          "BackupPlanName": "plan-name",
          "Rules": [
            {
              "CompletionWindowMinutes": 120,
              "Lifecycle": {
                "DeleteAfterDays": 1
              },
              "RuleId": "<uuid:1>",
              "RuleName": "test-rule",
              "ScheduleExpression": "cron(0 1 ? * * *)",
              "ScheduleExpressionTimezone": "America/Los_Angeles",
              "StartWindowMinutes": 60,
              "TargetBackupVaultName": "vault-name"
            }
          ]
        },
        "BackupPlanArn": "arn:<partition>:backup:<region>:111111111111:backup-plan:<backup-plan-id:1>",
        "BackupPlanId": "<backup-plan-id:1>",
        "CreationDate": "<datetime>",
        "VersionId": "<version-id:1>",
        "ResponseMetadata": {
          "HTTPHeaders": {},
          "HTTPStatusCode": 200
        }
      },
      "delete-backup-plan": "An error occurred (ResourceNotFoundException) when calling the GetBackupPlan operation: Failed reading Backup plan with provided version"
    }
  }
}
